scrapers/SARJ-LLC.py

```diff
@@ -89,16 +147,35 @@ def scrape_gallery(baseUrl, date, name):
     return map_media(data, studio, baseUrl)
 
 def map_model(baseUrl, model):
+    tags = list(map(lambda t: {'Name'}, model['tags']))
+
+    def add_tag(key, format):
+        nonlocal tags
+        if key in model and model[key] != "":
+            tags.append({
+                'Name': format.format(model[key])
+            })
+
+    add_tag('hair', '{} hair')
+    add_tag('pubicHair', '{} pussy')
+    add_tag('eyes', '{} eyes')
+    add_tag('breasts', '{} breasts')
+
     return {
         'Name': model['name'],
         'Gender': model['gender'].upper(),
         'URL': f"{baseUrl}{model['path']}",
         'Ethnicity': model['ethnicity'],
         'Country': model['country']['name'],
-        'EyeColor': model['eyes'],
+        'HairColor': model['hair'].capitalize(),
+        'hair_color': model['hair'].capitalize(),
+        'EyeColor': model['eyes'].capitalize(),
+        'eye_color': model['eyes'].capitalize(),
```

Comment:
As mentioned above the `hair_color` and `eye_color` are what we expect in the  json output.
EyeColor and HaiColor are not needed/used.
It seems a 
```python
        'Details': model['biography'],
```
can also be added to the performer data.

Reply:
Was a bit unsure about this, and I myself don't feel the need fot it, but added it now.

Reply:
Yeah i dont think i'll actually use sars for a performer scraper but its there if anyone wants to. The same goes with some of the fields i guess. The user can decide for him/herself what he wants to add when the scraper dialog appears.